File: Components/WinForms/CS/RibbonSimplePad/frmSplashScreen.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraSplashScreen;

namespace DevExpress.XtraBars.Demos.RibbonSimplePad {
    public partial class frmSplashScreen : SplashScreen {
        public frmSplashScreen() {
            InitializeComponent();
            pictureEdit2.Image = DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("DevExpress.XtraEditors.SplashScreen.DemoSplashScreen.png", typeof(DemoSplashScreen).Assembly);
            this.labelControl1.Text = AssemblyInfo.AssemblyCopyright;
        }

        #region Overrides

        public override void ProcessCommand(Enum cmd, object arg) {
            base.ProcessCommand(cmd, arg);
        }

        #endregion

        public enum SplashScreenCommand {
        }
    }
}

File: Components/WinForms/VB/RibbonSimplePad/frmSplashScreen.vb
Imports DevExpress.XtraSplashScreen

Public Class frmSplashScreen
    Sub New()
        InitializeComponent()
        Me.pictureEdit2.Image = DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("DevExpress.XtraEditors.SplashScreen.DemoSplashScreen.png", GetType(DemoSplashScreen).Assembly)
        Me.labelControl1.Text = AssemblyInfo.AssemblyCopyright
    End Sub
    Public Overrides Sub ProcessCommand(ByVal cmd As System.Enum, ByVal arg As Object)
        MyBase.ProcessCommand(cmd, arg)
    End Sub
    Public Enum SplashScreenCommand
        SomeCommandId
    End Enum
End Class
